packages/synapse-react-client/src/mocks/contribReqFormSchema.json

```diff
@@ -196,9 +196,17 @@
           "type": "boolean",
           "title": "Can data be released under the AMP-AD Knowledge Portal terms of use?",
           "default": false,
-          "enum": [true, false],
-          "enumNames": ["Yes", "No"],
```

Comment:
Support for non-standard `enumNames` property was removed in RJSF v6. I made sure to migrate production schemas for the portals using RJSF.